en/config/safety.md

```diff
@@ -219,6 +223,18 @@ The relevant parameters are shown below:
 | [COM_OF_LOSS_T](../advanced_config/parameter_reference.md#COM_OF_LOSS_T)   | Delay after loss of offboard connection before the failsafe is triggered.                                         |
 | [COM_OBL_RC_ACT](../advanced_config/parameter_reference.md#COM_OBL_RC_ACT) | Failsafe action if RC is available: Position mode, Altitude mode, Manual mode, Return mode, Land mode, Hold mode. |
 
+### Flight Time Failsafe
```

Comment:
FYI @sfuhrer Little odd to have two mechanisms for deciding on max flight time return - one estimated and one determined by you. Unless these are actually the trigger time and action that have been miss-documented?

Reply:
Those are two different things, one (COM_FLT_TIME_MAX) is to limit the flight time, and the other is a battery failsafe action to actually make sure you reach your landing point before you're out of juice.

Reply:
I would document COM_FLTT_LOW_ACT as part of the battery failsafes.

Reply:
@sfuhrer Thanks, done in https://github.com/PX4/PX4-user_guide/pull/3320

Note I chose not to do this originally because the separation in safety doc was between QGC failsafes and "Failsafes you have to set with parameters" and this crossed both. Also because the low level setting was quite distinct.

However as QGC safety view omits so much decided to make this feature centric instead, and have moved this as you suggested. It is an adjacent section to the old settings, both grouped under "Battery Failsafes". Works much better IMO.